apps/ehr/src/telemed/features/appointment/ReviewTab/ReviewAndSignButton.tsx

```diff
@@ -74,6 +74,8 @@ export const ReviewAndSignButton: FC<ReviewAndSignButtonProps> = ({ onSigned })
     return appointmentAccessibility.status === TelemedAppointmentStatusEnum.complete;
   }, [css, inPersonStatus, appointmentAccessibility.status]);
 
+  const isProviderStatus = useMemo(() => inPersonStatus === 'provider', [inPersonStatus]);
```

Comment:
For such cases when the calculation logic is very simple and the value is a primitive, there's no point in using memoization.